app/models/task.rb

```diff
@@ -0,0 +1,23 @@
+class Task < ActiveRecord::Base
+
+  belongs_to :person
+
+  STATUS = {is_complete: ["uncomplete", "complete"], not_complete: ["complete", "uncomplete"]}
+
+  def self.all_tasks
```

Comment:
I'm not sure why you needed this to be separate -- couldn't you just use Task.all wherever you would write Task.all_tasks?